plugin/src/sidebar.ts

```diff
@@ -14,11 +14,12 @@ let addedDatasets: string | undefined = undefined;
 
 let rawCatalog: CatalogRawItem[] = [];
 
-const doNotShowWelcome = true; // Make it `let doNotShowWelcome: boolean = false`, and then modify based on storage value when Storage API available
+let doNotShowWelcome = false; // Make it `let doNotShowWelcome: boolean = false`, and then modify based on storage value when Storage API available
 
 // let isMobile: boolean;
 
 reearth.ui.show(html, { extended: true });
+reearth.modal.show(welcomeScreenHtml, { width: 1811, height: 959 });
```

Comment:
If you look at the design, you'll want to set the width and height based on the viewport (the X button is in the top right corner). 
Also, you will only want to show this modal IF there is doNotShowWelcome value in the storage. Having it here means it'll show every time the page is loaded. It also seems like there is no logic to check if there is a doNotShowWelcome value in storage, and if there is, and it is true, to NOT show the modal.

Reply:
sorry it is for test purpose , removed

Reply:
2e7e5ada980318f9de64a960bc7f473718816a86

Reply:
for viewport 3316a3a6b9f898adf24261ce66d0e2f4b61d8fcc